store/src/main/java/com/dropbox/android/external/store4/MemoryPolicy.kt

```diff
@@ -3,18 +3,15 @@ package com.dropbox.android.external.store4
 import java.util.concurrent.TimeUnit
 
 /**
- * MemoryPolicy holds all required info to create MemoryCache and
- * [NoopPersister]
+ * MemoryPolicy holds all required info to create MemoryCache
  *
  *
- * This class is used, in order to define the appropriate parameters for the MemoryCache
+ * This class is used, in order to define the appropriate parameters for the Memory [com.dropbox.android.external.cache3.Cache]
```

Comment:
for another PR: should we clean up the package/coordinates before we release something like com.dropbox.store?

Reply:
This is the cache that will be rewritten, I wanted to leave it as cache3 so that @ychescale9 impl can be released as cache4.

Reply:
I meant for store in general not just for cache (move to package `com.dropbox.store` . Again if we were to do it it's for another PR